packages/driver/src/cy/commands/actions/select.js

```diff
@@ -246,7 +256,9 @@ module.exports = (Commands, Cypress, cy) => {
 
             change.initEvent('change', true, false)
 
-            return options.$el.get(0).dispatchEvent(change)
+            options.$el.get(0).dispatchEvent(change)
+
+            return undefined
```

Comment:
Not returning at all returns undefined, so this seems superfluous.

Reply:
Yeah, I guess I prefer being specific about intended returns.